### PR TITLE
ENG-5026 plugin error handling

### PR DESCRIPTION
**Pull request by antromeo**
Wait for ENG-5006

### CODE REVIEW

File: src/commands/convert.ts [hunk 1]
```diff
@@ -29,6 +30,25 @@ const DESCRIPTOR_V5_FOUND = 'The Bundle is already a v5'
 const MANUALLY_STEPS_DESCRIPTION = 'MANUALLY STEPS TO DO'
```
Comment (avdev4j) [inline src/commands/convert.ts]:
```suggestion
const MANUAL_STEPS_DESCRIPTION = 'MANUAL STEPS TO DO'
```

File: src/commands/convert.ts [hunk 2]
```diff
@@ -29,6 +30,25 @@ const DESCRIPTOR_V5_FOUND = 'The Bundle is already a v5'
...
 const SERVICE_FILES_DESCRIPTION = 'CONVERSION OF SERVICE FILES'
 const PLATFORM_FILES_DESCRIPTION = 'CONVERSION OF PLATFORM FILES'
+const MANUALLY_STEPS = [
```
Comment (avdev4j) [inline src/commands/convert.ts]:
```suggestion
const MANUAL_STEPS = [
```

File: src/commands/convert.ts [hunk 3]
```diff
@@ -29,6 +30,25 @@ const DESCRIPTOR_V5_FOUND = 'The Bundle is already a v5'
...
+  `\n${MANUALLY_STEPS_DESCRIPTION}\n`,
```
Comment (avdev4j) [inline src/commands/convert.ts]:
```suggestion
  `\n${MANUAL_STEPS_DESCRIPTION}\n`,
```

File: src/commands/convert.ts [hunk 4]
```diff
@@ -132,19 +162,10 @@ export default class Convert extends Command {
...
+    register.push(...getPlatformDescription(platformDescReport))
     CliUx.ux.action.stop()
 
-    register.push(
-      `\n${MANUALLY_STEPS_DESCRIPTION}`,
-      'Add the source files in new folders microservices, microfrontends',
-      'Check that you have in your docker compose files a service name corresponding to the service filename',
-      'If you want to change the bundle name, edit the folder name and entando.json\n'
-    )
+    register.push(...MANUALLY_STEPS)
```
Comment (avdev4j) [inline src/commands/convert.ts]:
```suggestion
    register.push(...MANUAL_STEPS)
```

**Review by avdev4j — CHANGES_REQUESTED** [review]

**Comment by avdev4j** [comment]
I tried to remove the plugin descriptor in my tested project (v1 bundle) but keep the entry in the main descriptor and no errors have been raised. Also, the output messages in the console don't match the examples included in the ticket. Any idea @antromeo ?

**Comment by antromeo** [comment]
> I tried to remove the plugin descriptor in my tested project (v1 bundle) but keep the entry in the main descriptor and no errors have been raised. Also, the output messages in the console don't match the examples included in the ticket. Any idea @antromeo ?

yes, the tool shouldn't break, if there is a plugin with a processing error, only the plugin with error will be skipped.
if you want to see more details in the console, you need to enable debug mode or check the log file

**Review by avdev4j — APPROVED** [review]

**Pull request merged**